public/components/common/charts/visualizations/legend.tsx

```diff
@@ -17,7 +17,7 @@ type ChartLegendProps = {
  */
 export function ChartLegend({ data }: ChartLegendProps) {
   const list = data.map(({label, labelColor, value, ...rest}, idx) => ({
-    label: `${label} (${value})`,
+    label: <div style={{fontSize: '0.793rem'}}>{`${label} (${value})`}</div>,
```

Comment:
**nitpick**: The `EuiListGroup` component has a property to modify the size of all `EuiListItem`. You could use the property instead of the proposed solution.

Example:
```tsx
return (
  <EuiListGroup
    className="chart-legend"
    listItems={list}
    color='text'
    size='xs'
    flush />
)
```